src/components/PageTemplate/Navbar.jsx

```diff
@@ -74,7 +75,7 @@ const Links = () => (
         <Button bg="transparent" m={MARGIN} as={Link} to="/">Home</Button>
         <Button bg="transparent" m={MARGIN} as={Link} to="/watchlist">Watchlist</Button>
         <Button bg="transparent" m={MARGIN} as={Link} to="/news">News</Button>
-        <Button bg="transparent" m={MARGIN} as={Link} to="/about">About</Button>
+        <Button bg="transparent" m={MARGIN} as={Link} to="/plans">Plans</Button>
```

Comment:
Would be a better idea to not repeat css configs and have a customNavbarButton as a component